building the git version in MSYS2

How do I build the git version of Premake using `Bootstrap.mak` in MSYS2? The closest target platform is `mingw` but the following command fails:
```
user@host MINGW64 /c/projects/premake5/premake-5.0.0-beta1-src
$ make -f Bootstrap.mak  mingw
if exist .\bin rmdir /s /q .\bin
/bin/sh: -c: line 2: syntax error: unexpected end of file
make: *** [Bootstrap.mak:80: windows-clean] Error 2
```

Looks like it fails because of the windows-style commands in the `windows-clean` target. Shouldn't the `mingw-clean` inherit from the `nix-clean` instead of current `windows-clean`?
https://github.com/premake/premake-core/blob/0c6711170c60c208e15573a749b4fd21d5b28f6a/Bootstrap.mak#L84

I know that Premake is in the MSYS2 packages and that I can install it from there, but still.

--- a/.github/workflows/ci-workflow.yml
+++ b/.github/workflows/ci-workflow.yml
@@ -70,3 +70,42 @@ jobs:
       with:
         name: premake-windows-${{ matrix.platform }}
         path: bin\${{ matrix.config }}\
+  mingw:
+    runs-on: windows-2022
+    strategy:
+      matrix:
+        config: [debug, release]
+        msystem: [mingw32, mingw64]
+        include:
+          - platform: x86
+            msystem: mingw32
+          - platform: x64
+            msystem: mingw64
+    defaults:
+      run:
+        shell: msys2 {0}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: 'Setup MSYS2'
+      uses: msys2/setup-msys2@v2
+      with:
+        msystem: ${{ matrix.msystem }}
+        update: true
+        install: >-
+          git
+          make
+        pacboy: >-
+          toolchain:p
+    - name: Build
+      run: make -f Bootstrap.mak mingw PLATFORM=${{ matrix.platform }} CONFIG=${{ matrix.config }}
+    - name: Test
+      run: bin/${{ matrix.config }}/premake5.exe test --test-all
+    - name: Docs check
+      run: bin/${{ matrix.config }}/premake5.exe docs-check
+    - name: Upload Artifacts
+      if: matrix.config == 'release'
+      uses: actions/upload-artifact@v2
+      with:
+        name: premake-${{ matrix.msystem }}-${{ matrix.platform }}
+        path: bin\${{ matrix.config }}\

--- a/Bootstrap.mak
+++ b/Bootstrap.mak
@@ -81,14 +81,14 @@ windows-clean:
 	$(SILENT) if exist .\build rmdir /s /q .\build
 	$(SILENT) if exist .\obj rmdir /s /q .\obj
 	
-mingw-clean: windows-clean
+mingw-clean: nix-clean
 	
 mingw: mingw-clean
-	if not exist build\bootstrap (mkdir build\bootstrap)
+	mkdir -p build/bootstrap
 	$(CC) -o build/bootstrap/premake_bootstrap -DPREMAKE_NO_BUILTIN_SCRIPTS -I"$(LUA_DIR)" -I"$(LUASHIM_DIR)" $(SRC) -lole32 -lversion
 	./build/bootstrap/premake_bootstrap embed
 	./build/bootstrap/premake_bootstrap --arch=$(PLATFORM) --os=windows --to=build/bootstrap --cc=mingw gmake2
-	$(MAKE) -C build/bootstrap config=$(CONFIG)_$(PLATFORM:x86=win32)
+	$(MAKE) -C build/bootstrap -j`getconf _NPROCESSORS_ONLN` config=$(CONFIG)_$(PLATFORM:x86=win32)
 
 macosx: osx
 

